canarytokens/channel_output_email.py

```diff
@@ -295,6 +295,7 @@ def format_report_html(
         readable_type = readable_token_type_names[details.token_type]
         BasicDetails = details.dict()
         BasicDetails["readable_type"] = readable_type
+        BasicDetails["type"] = str(details.token_type)
```

Comment:
are you sure this `str(details.token_type)` gives you what you expect? im not sure this gives you the enum value. you can also just do `details.token_type.value` which is what i suspect you want here

Reply:
also can we make the field `token_type` instead of `type`

Reply:
> are you sure this str(details.token_type) gives you what you expect?
The __str__function in the TokenTypes class is called when a TokenTypes object is passed to str() - and it returns TokenType.value. But that's a roundabout way of just using details.token_type.value. I'll change that.

There's already a key in BasicDetails called `token_type`. Should I make it token_type_value?

Reply:
what do you get if you create that Enum and the terminal and call str() on one of it's items?

`token_type_value` is fine